soc/arm/nxp_kinetis/k6x/Kconfig.defconfig.mk64f12

```diff
@@ -11,6 +11,13 @@ config SOC
 config ADC_MCUX_ADC16
 	default y
 	depends on ADC
+if DAC
+
+config DAC_MCUX_DAC
+	default y
```

Comment:
instead of wrapping with `if DAC..endif`, just add `depends on DAC`

Reply:
Fixed.